lib/internal/Magento/Framework/Cache/Config/Reader.php

```diff
@@ -24,7 +30,7 @@ class Reader extends \Magento\Framework\Config\Reader\Filesystem
      * @param string $domDocumentClass
      * @param string $defaultScope
      */
-    public function __construct(
+    public function __construct( //phpcs:ignore Generic.CodeAnalysis.UselessOverridingMethod
         \Magento\Framework\Config\FileResolverInterface $fileResolver,
         \Magento\Framework\Cache\Config\Converter $converter,
         \Magento\Framework\Cache\Config\SchemaLocator $schemaLocator,
```

Comment:
Import classes

Reply:
Hello @lbajsarowicz 
Thank you for your remark.
I have fixed it. Please check.
Thank you in advance.